### ISSUE TITLE
PC0030 - explicit SetLoadFields()

### ISSUE THREAD

**Issue by AnsBarMIT** [ALCops/Analyzers#261]
We have some code that copies records from a local variable to a temporary table.
According to the documentation, partial records should not be used here [(MS Learn)](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-partial-records):

>For performance reasons, it's not recommended to use partial records on a record that will do inserts, deletes, renames, field transfers, or copies to temporary records.

Currently a `#pragma` would have to be added to disable the PC0030 diagnostic.

Maybe an explicit `SetLoadFields()` without parameters (with an additional explaining comment?) call could also be used?
More of a style thing than a bug, but to me the explicit `SetLoadFields()` seems clearer than to pragma away the diagnostic (unless i am missing some side-effect).

```al
// more logic
LocalRecord.SetLoadFields(); // could be used to explicitly indicate that loading all fields is wanted here?
if LocalRecord.FindSet() then // PC0030 is reported here
    repeat
        // more logic
        TempRecord := LocalRecord;
        TempRecord .Insert(false);
    until LocalRecord.Next() = 0;
```

**Comment by AnsBarMIT** [comment]
Just saw #260, will comment there.
